diff --git a/PROJECT_MANIFEST.sha256 b/PROJECT_MANIFEST.sha256
index 5ac637b..42fc784 100644
--- a/PROJECT_MANIFEST.sha256
+++ b/PROJECT_MANIFEST.sha256
@@ -89,7 +89,7 @@ d42315abb2f0142200bd995cef24d52cfc4c903afe36c2ae9f018e78a374defb  ./app/src/main
 fcdcd918517d7509b866a73787203ac057b402aabc2794bae97ea8413bc0d15c  ./app/src/main/java/dev/securemesh/commander/navigation/RootSessionLogic.kt
 3c0e61230f478c72b9bb76e3cdf0deb769d443a6cdceb590bcf1cdf481a3ff3a  ./app/src/main/java/dev/securemesh/commander/navigation/RootViewModel.kt
 455bdbca1c1d63d0d26e011c03bc375e6061123e1f289e1e4eb582dc4d0f1e29  ./app/src/main/java/dev/securemesh/commander/navigation/SecureMeshRoot.kt
-b5745a96f2e7a5b5c291200dc2e3bd74d68b73ddc69069c63af523d0c98186b7  ./app/src/main/res/values-v27/themes.xml
+108769ba75053ec22b4744a944ca5f2ea93c48dc5171965e06166993b8b976f7  ./app/src/main/res/values-v27/themes.xml
 9d0730f0c8c0a73892545750a6e5458071c6b6bed84b51b7a29238ac70122bf7  ./app/src/main/res/values/strings.xml
 2d9f2183e5e1b2c43d327f4935cb9cb7f1ec8cba0d7e6195d5df3123562fc563  ./app/src/main/res/values/themes.xml
 2c5c697984f4c52b526337ab61113f5843e0bfc6af4722374916ffe6da38dd9a  ./app/src/main/res/xml/backup_rules.xml
diff --git a/app/src/main/res/values-v27/themes.xml b/app/src/main/res/values-v27/themes.xml
index df56751..f82e4cf 100644
--- a/app/src/main/res/values-v27/themes.xml
+++ b/app/src/main/res/values-v27/themes.xml
@@ -1,5 +1,12 @@
 <resources>
-    <style name="Theme.SecureMesh">
+    <style name="Theme.SecureMesh" parent="android:style/Theme.Material.NoActionBar">
+        <item name="android:fontFamily">sans</item>
+        <item name="android:windowLightStatusBar">false</item>
         <item name="android:windowLightNavigationBar">false</item>
+        <item name="android:windowBackground">#030A10</item>
+        <item name="android:statusBarColor">#030A10</item>
+        <item name="android:navigationBarColor">#030A10</item>
+        <item name="android:colorAccent">#35D9FF</item>
+        <item name="android:windowActionModeOverlay">true</item>
     </style>
 </resources>
